doc/rst/technotes/initTypeAlias.rst

```diff
@@ -0,0 +1,161 @@
+
+.. _readme-initTypeAlias:
+
+.. default-domain:: chpl
+
+==================================================
+Invoking Initializers with a Generic Instantiation
+==================================================
+
+Overview
+--------
+
+Initializing a variable with a generic type has historically required users to
+pass each ``type`` and ``param`` value to the initializer as arguments. This
+can be a tedious and repetitive process if the user is initializing many
+variables of the exact same type. Some users have expressed interest in support
```

Comment:
Let's just say something like "One might wish to use type aliases to reduce ...."